contrib/planb-objsync.py

```diff
@@ -1121,7 +1380,7 @@ def _add_new_record_download(self, record, container, path):
                     'Got exception %r/%s during unlink %s', e2, e2, path)
             raise e
         except Exception as e:
-            log.warning(
+            log.exception(
```

Comment:
Why the log-level change?

This looks like the part where we only warn when new files in the listing have disappeared. This can happen when we first create a big listing (takes dozens of minutes) and then fetch the files. In a highly mutating environment, we can find many files which are gone or have different sizes/timestamps by that time.

- For Swift/S3 download failures (and then md5 failures), I could go with the Warning. 
- For the NotImplementedError, I agree that that is an exception.

Reply:
Wanted to see the traceback, fixed.